foxy/distribution.yaml

```diff
@@ -3748,7 +3748,7 @@ repositories:
       tags:
```

Comment:
We should also update the source an doc version to point to `foxy-devel`. That can be done in this PR or a follow-up.

Reply:
huh, I thought it was going to be updated automatically when `bloom-release`.
I will fix it here

Reply:
I guess it can update it automatically. Did you run bloom with `--edit`?

Reply:
Done in https://github.com/ros/rosdistro/pull/28491/commits/bec23a28a848cb6a76b29a17504651ca7f745600

Reply:
> I guess it can update it automatically. Did you run bloom with `--edit`?

yeap, it updated the tracks automatically and not this